foundations/javascript_basics/fundamentals-1.md

```diff
@@ -41,8 +41,17 @@ The simplest way to get started is to simply create an HTML file with the JavaSc
 
 Save and open this file up in a web browser (you can use Live Server to do this!) and then <span id="access-devTools-console">open up the browser's console by right-clicking on the blank webpage and selecting "Inspect" or "Inspect Element".  In the 'Developer Tools' pane find and select the 'Console' tab</span>, where you should see the output of our `console.log` statement.
 
+
 > <span id="console-log">`console.log()` is the command to print something to the developer console in your browser. You can use this to print the results from any of the following articles and exercises to the console.</span> We encourage you to code along with all of the examples in this and future lessons.
 
+Another way to include is through an external script. This is very similar to linking external CSS docs to your website. 
+
+~~~html
+  <script src="javascript.js"></script>
+~~~
+
+Javascript files have the extension `.js` similar to `.css` for stylesheets. External javascript sheets are used for more complex scripts. Simple scripts such as logging to the console can stay within the script tag.
```

Comment:
```suggestion
JavaScript files have the extension `.js` similar to `.css` for stylesheets. External JavaScript files are used for more complex scripts.
```

I suggest these changes because I think that referring to JS files as sheets could cause confusion, and I don't think the second sentence is really needed since adding JS directly into a script tag for simple code was already mentioned. Also because JS should *technically* be written in pascal case.